### PR TITLE
Issue/79 | Add logger

### PR DESCRIPTION
**Pull request by 0xBEEFCAF3**
fix https://github.com/chaincase-app/nolooking/issues/79

### CODE REVIEW

**Comment by 0xBEEFCAF3** [comment]
@DanGould Cab you rerun the test job please?
It looks like the failure is unrelated to the changes in this PR
```
thread 'integration::test' panicked at 'called `Result::unwrap()` on an `Err` value: ConnectError { internal: ReadFile { file: "/tmp/.tmpMbMual/peer-tls.cert", error: Os { code: 2, kind: NotFound, message: "No such file or directory" } } }',
```

**Comment by 0xBEEFCAF3** [comment]
@DanGould ready for review

File: README.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ lnd_macaroon_path="/home/dan/.lnd/data/chain/bitcoin/mainnet/admin.macaroon"
    - Lines starting with `lnd_` specify your connection to your bitcoin node.
 
    - You will be able to view the nolooking site on `bind_port` and the payjoin endpoint will be `endpoint` (e.g. can be a domain).
-
-4. Run with `nolooking --conf nolooking.conf`
-5. Visit Nolooking on [127.0.0.1](http://127.0.0.1:3000) and queue some bitcoin channels.
-6. Generate the QR code, pay it or share it! Once a payjoin transaction has enough confirmations, your new lightning channels will be established and you can move your sats over the lightning nework!
+4. (Optional) nolooking uses [env_logger](https://docs.rs/env_logger/latest/env_logger/) to log messages varying priority. Use `RUST_LOG` env var to set your log priority. Valid options are error, warn, info, debug, trace
```
Comment (DanGould) [inline README.md]: On principle we avoid options whenever possible. `env_logger` affords us sane defaults: https://github.com/rust-cli/env_logger/issues/47#issuecomment-607475404

File: src/http.rs [hunk 2]
```diff
@@ -81,7 +82,7 @@ async fn serve_public_file(path: &str) -> Result<Response<Body>, HttpError> {
 }
 
 async fn handle_pj(scheduler: Scheduler, req: Request<Body>) -> Result<Response<Body>, HttpError> {
-    dbg!(req.uri().query());
+    info!("{:?}", req.uri().query());
```
Comment (DanGould) [inline src/http.rs:85]: nit: If display isn't implemented, this is probably debug, not info

**Review by DanGould — COMMENTED** [review]
Thanks for grabbing this chore and implementing it really well.

I suggest we change the option to a default. CARGO_LOG envvar is set on debug by default when you run a default build anyhow, and in production it will default to info level.

**Pull request merged**